compiler/runtime/Runtime.hpp

```diff
@@ -413,6 +413,7 @@ typedef struct TR_RelocationRecordInformation {
    uintptr_t data2;
    uintptr_t data3;
    uintptr_t data4;
+   uintptr_t data5;
```

Comment:
Could you please describe in brief why the new field is being added ? The names of these variables are not descriptive enough for me to assess what they are used for eventually. Maybe update the commit message too.